pkg/fswatcher/fs_watcher.go

```diff
@@ -16,40 +16,49 @@ package fswatcher
 
```

Comment:
I suggest renaming fs_watcher*.go files to plain watcher*.go and creating new interface/impl/tests in fswatcher*.go